index.html

```diff
@@ -0,0 +1,24 @@
+<html>
```

Comment:
What's this file for? A demo? If so, I'd consider a demos or examples folder.

Reply:
Also just for me to test out the components -- so I guess a demo. I'll move it into a folder.